### ISSUE TITLE
[Accessibility][Usable - Property Pane_Ask a question (Bot Asks)]: At prompt edit field, error message appears on first error for short time.

### ISSUE THREAD

**Issue by ashish315** [microsoft/BotFramework-Composer#2112]
**User Experience** 
Users who rely on AT will face difficulty and easily confused if the error disappears after sometime.

**Test Environment**
Browser: Microsoft Edge Canary Version: 82.0.425.0 (Official build)
OS build: 2004 (19564.1005)
Screen Reader: Narrator
URL: [Bot Framework Composer](http://localhost:3000/dialogs/Main)
[**Prerequisite Here**](https://github.com/microsoft/BotFramework-Composer)

**Repro Steps:** 
1. Open the above mentioned URL in EDGE and turn on narrator.
2. Navigate to already created dialog tree item and activate it. Such as- Greetings, Actions, EndDialog etc.
3. Click on "+" and select 'Ask a question'. Then select any of the available options except Oauth login.
4. Navigate to property pane appeared at right side of the screen.
5. Now, navigate to 'Prompt' edit field and delete all the content & press space.
6. Observe the issue.

**Actual:** 
On entering blank data in the prompt edit field, an error message flashes for a shorter time where as on pressing space twice the error message appears permanently which is expected behavior.

**Expected:**
The error message should be appeared and stays at the first place when the incorrect or blank user input is provided

**Note:**
Same issue is observed for all the Property Pane which have prompt or any syntax edit fields appeared at right side of the screen.

**MAG Reference:** [MAG](https://microsoft.sharepoint.com/teams/msenable/_layouts/15/Doc.aspx?sourcedoc=%7BD0F24E02-1D9C-4DA2-8F5B-2A3C1176F3BF%7D&file=Guideline%2017%20A%20-%20Contrast%20(Minimum)%20When%20Conveying%20Information.docx&action=edit&mobileredirect=true)

**Attachment:**
[Usable_error message disappears.zip](https://github.com/microsoft/BotFramework-Composer/files/4261673/Usable_error.message.disappears.zip)

**Comment by beyackle** [comment]
This appears to be fixed already; I couldn't reproduce it.

**Comment by corinagum** [comment]
@ashish315 and @awalia13, can we get a reproduction as fixed from your team? Thank you!

**Comment by ashish315** [comment]
As checked and verified on below environment, the issue is fixed now. So, we are closing the issue.

**Test Environment**
Browser: Microsoft Edge Dev {Version 83.0.467.0 (Official build) dev (64-bit)}
OS build: 2004 (19588.1000)
Screen Reader: Narrator
URL: [Bot Framework Composer](http://localhost:3000/dialogs/Main)
[**Prerequisite Here**](https://github.com/microsoft/BotFramework-Composer)
